daemon/src/maker_inc_connections.rs

```diff
@@ -106,39 +129,32 @@ impl Actor {
             .map_ok(move |msg| FromTaker { taker_id, msg })
             .map(forward_only_ok::Message);
 
-        let (out_msg_actor_address, mut out_msg_actor_context) = xtra::Context::new(None);
-
-        let (forward_to_cfd, forward_to_cfd_fut) =
+        let (forward_only_ok, forward_only_ok_fut) =
             forward_only_ok::Actor::new(self.taker_msg_channel.clone_channel())
```

Comment:
I can already see that I meant for the `forward_only_ok::Actor` to forward to the `maker_inc_connections::Actor` (i.e. this one), so this is wrong.

Reply:
I am not sure if we still need `forward_only_ok` now.

Reply:
I too wasn't sure, but it is useful to track when the reading end of the stream fails: the actor will go down and we can react accordingly by dropping all the tasks related to that taker ID.